babel.config.js

```diff
@@ -2,7 +2,14 @@ module.exports = {
   env: {
     test: {
       presets: [
-        ["@babel/preset-env"]
+        [
+          "@babel/preset-env",
+          {
+            "targets": {
+              "node": "current"
+            }
+          }
```

Comment:
This resolves an issue where running Jest results in `ReferenceError: regeneratorRuntime is not defined`.